sui/its.js

```diff
@@ -0,0 +1,220 @@
+const { Command } = require('commander');
+const { Transaction } = require('@mysten/sui/transactions');
+const { bcs } = require('@mysten/sui/bcs');
+const { bcsStructs } = require('@axelar-network/axelar-cgp-sui');
+const { loadConfig, saveConfig } = require('../common/utils');
+const {
+    getBcsBytesByObjectId,
+    addBaseOptions,
+    addOptionsToCommands,
+    getUnitAmount,
+    getWallet,
+    printWalletInfo,
+    broadcast,
+} = require('./utils');
+const { ethers } = require('hardhat');
+const {
+    utils: { arrayify },
+} = ethers;
+
+async function sendTokenTransfer(keypair, client, contracts, args, options) {
```

Comment:
this script should interact with ITS directly instead of the example, similar to `gateway.js`, and `evm/its.js`. It's meant to be an operational script (allows using ITS functionality, but also operations like setting trusted addresses for e.g. Perhaps this can be renamed to `its-app.js` or `its-example.js` or so to distinguish from core ITS

Reply:
renamed to `its-example`